src/coreclr/pal/src/arch/loongarch64/signalhandlerhelper.cpp

```diff
@@ -52,7 +52,7 @@ void ExecuteHandlerOnCustomStack(int code, siginfo_t *siginfo, void *context, si
 
     // Build fake stack frame to enable the stack unwinder to unwind from signal_handler_worker to the faulting instruction
     // pushed RA
-    *--sp = (size_t)MCREG_Pc(ucontext->uc_mcontext);
+    *--sp = (size_t)ucontext->uc_mcontext.__pc;
```

Comment:
Can you please keep using the `MCREG_Pc` macro instead so that the access to Fp and Pc is done the same way here like we do that for other architectures? The macro seems to be defined to read the same thing:
```C++
#define MCREG_Pc(mc)      ((mc).__pc)
```

Reply:
Ok, thanks